nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
+The number of messages to [prefetch](http://www.rabbitmq.com/consumer-prefetch.html) when consuming messages from the broker.
 
-Defaults: The number of configured threads for the transport(as of v2.1)
+Default: The number of configured threads for the transport(as of v2.1)
 
-Versions: 3 and below. In Versions 4 and above `EndpointConfiguration.LimitMessageProcessingConcurrencyTo` can be used instead. See [Throughput Throttling](/samples/throttling/) sample.
+Versions: 3 and below.
 
 
 #### UsePublisherConfirms
 
 Controls if [publisher confirms](https://www.rabbitmq.com/confirms.html) should be used. 
 
-Defaults: `true`
+Default: `true`
 
 
 #### MaxWaitTimeForConfirms
 
-How long the client should wait for publisher confirms if enabled. 
+How long the client should wait for publisher confirms, if enabled.
+
+Default: `30` seconds.
 
-Defaults: `30` seconds.
+Versions: 3 and below.
 
 
 #### RetryDelay
 
-The time to wait before trying to reconnect to the broker if connection is lost.
+The time to wait before trying to reconnect to the broker if the connection is lost.
 
-Defaults: `10` seconds
+Default: `10` seconds
 
 
 #### UseTls
 
-Indicates if the connection should be done using TLS. 
+Indicates if the connection to the broker should be secured with [TLS](/nservicebus/rabbitmq/configuration-api.md#specifying-the-connection-string-transport-layer-security-support).
 
-Default: If `Port` is not specified and `UseTls` is enabled the port will default to `5671`.
+Default: `false`
 
 Versions: 4 and above
 
 
 #### CertPath
 
-The certificated path that should be used when using TLS.
+The file path to the client authentication certificate when using [TLS](/nservicebus/rabbitmq/configuration-api.md#specifying-the-connection-string-transport-layer-security-support)
 
 Versions: 4 and above
 
 
 #### CertPassphrase
 
-The certificate password.
+The password for the client authentication certificate specified in `CertPath`
 
 Versions: 4 and above
 
 
-## Specifying the connection string
-
-To use a custom name for the connection string use:
-
-snippet:rabbitmq-config-connectionstringname
+NOTE: For debugging purposes, it can be helpful to increase the `RequestedHeartbeat` and `DequeueTimeout` settings like this:
 
-To specify the connection string in code:
+snippet:rabbitmq-connectionstring-debug
 
-snippet:rabbitmq-config-connectionstring-in-code
 
-For debugging purposes, increase the `RequestedHeartbeat` and `DequeueTimeout` like this:
+## Callback support
 
-snippet:rabbitmq-connectionstring-debug
+When scaling out an endpoint using the RabbitMQ transport, each instance of the endpoint will consume messages from the same shared broker queue. This works in most scenarios because any instance can handle any incoming message. However, this is not true for callback messages, which need to be handled by the instance that requested the callback. 
```

Comment:
>However, this is not true for callback messages, which need to be handled by the instance that requested the callback. 

I think what needs to be highlighted is the fact that for Callbacks, it is not OK for any instance to handle the message that arrives in the queue, but that exact instance that requested the callback. Not sure that the above conveys that.

How about:
When scaling out an endpoint using the RabbitMQ transport, any of the endpoint instances can consume messages from the same shared broker queue. However, this default behavior can cause problems when dealing with callback messages because the reply messages for the callback needs to go to a particular instance of the endpoint, the same instance that requested the callback.